src/styles/blocks/calendar.scss

```
@@ -0,0 +1,40 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $calendar-width;
+  margin: 10% auto;
+  padding: $calendar-pd;
+  @for $index from 28 through $calendar-numberOf-day {
+    &--month-length--#{$index} >
+    .calendar__day:nth-last-child(-n + #{31 - $index}) {
```

Use scss here like below: 
```suggestion
    .&__day:nth-last-child(-n + #{31 - $index}) {
```